### PR TITLE
Lot 50/remove financial institution mutation

### PR DESCRIPTION
**Pull request by Webhst**

### CODE REVIEW

File: src/graphql/financial-institution/financial-institution.graphql [hunk 1]
```diff
@@ -46,4 +47,5 @@ type Query {
 }
 type Mutation {
   connectFinancialInstitution(input: ConnentFinancialInstitutionInput): ConnentFinancialInstitutionOutput
+  removeFinancialInstitution(input: Int): Boolean
```
Comment (brianschardt) [inline src/graphql/financial-institution/financial-institution.graphql]: Should have the input type  TemoveFinancialInstitutionInput I know its tedious but want to make sure everything is consistent.

Comment (Webhst) [reply src/graphql/financial-institution/financial-institution.graphql]: Okay going to refactor that

Comment (Webhst) [reply src/graphql/financial-institution/financial-institution.graphql]: done!

File: src/graphql/financial-institution/financial-institution.resolver.ts [hunk 2]
```diff
@@ -47,4 +48,15 @@ import { ReqUser, ConnentFinancialInstitutionInput, ConnentFinancialInstitutionO
         };
     }
 
+    @Mutation()
```
Comment (brianschardt) [inline src/graphql/financial-institution/financial-institution.resolver.ts]: Create a guard that only allows the user who owns the financial institution or a user who is an internalAdmin to remove the financial institution.

Comment (Webhst) [reply src/graphql/financial-institution/financial-institution.resolver.ts]: Working on that

Comment (Webhst) [reply src/graphql/financial-institution/financial-institution.resolver.ts]: done ! is working , i think is right what do you think ?

**Pull request merged**